aviary/mission/flops_based/ode/mission_ODE.py

```diff
@@ -144,15 +147,24 @@ def setup(self):
         # to the ODE with a special configure() method that promotes
         # all aircraft:* and mission:* variables to the ODE.
         external_subsystem_group = ExternalSubsystemGroup()
+        external_subsystem_group_solver = ExternalSubsystemGroup()
         add_subsystem_group = False
+        add_subsystem_group_solver = False
 
         for subsystem in self.options['external_subsystems']:
             subsystem_mission = subsystem.build_mission(
                 num_nodes=nn, aviary_inputs=aviary_options
             )
             if subsystem_mission is not None:
-                add_subsystem_group = True
-                external_subsystem_group.add_subsystem(
+
+                if subsystem.needs_mission_solver(aviary_options):
+                    add_subsystem_group_solver = True
+                    target = external_subsystem_group_solver
```

Comment:
external_subsystem_group_with_solver might be more clear? at first I thought we were adding the subsystem to the solver itself